staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -163,7 +163,27 @@ func getRequestOptions(req *http.Request, scope RequestScope, into runtime.Objec
 	return scope.ParameterCodec.DecodeParameters(query, scope.Kind.GroupVersion(), into)
 }
 
+// TODO: support the ability to watch for resource with options
```

Comment:
I'm a bit concerned that by adding a list operation without a corresponding watch operation that we'll encourage clients to do lists where they should do watches.